src/transformers/modeling_auto.py

```diff
@@ -116,6 +117,7 @@
     LongformerForTokenClassification,
     LongformerModel,
 )
+from .modeling_lxmert import LxmertForPreTraining, LxmertModel
```

Comment:
The question answering model wasn't added here as it has a very different API to other QA models.